component/prometheus/operator/podmonitors/operator.go

```diff
@@ -90,6 +90,8 @@ func (c *Component) Run(ctx context.Context) error {
 }
 
 // Update implements component.Component.
+// TODO(jcreixell): Initialize manager here so we can return errors back early to the caller.
+// See https://github.com/grafana/agent/pull/2688#discussion_r1152384425
```

Comment:
nit: please don't attach TODO comments as documentation comments on functions to prevent it showing up as part of the documentation, rather put TODO comments inside the body of the function (or type if adding a TODO to a type) instead